mud_files/easyio_device.json

```diff
@@ -0,0 +1,136 @@
+{
+  "ietf-mud:mud" : {
+    "mud-version" : 1,
+    "mud-url" : "https://digital-building.org/mud/easyio_device",
+    "last-update" : "2018-12-18T17:32:49.890Z",
+    "cache-validity" : 48,
+    "is-supported" : true,
+    "systeminfo" : "EasyIO BACnet device",
```

Comment:
Ideally we could have this be generic and not EasyIO specific... is the port 1876 a "special" easy-IO port? If so, then this should be fine, if it's some kind of generic standard then we should have this be a generic template!

Reply:
Yes, it is a specific to the EasyIO/Sedona framework. Unfortunately when we start looking into commissioning software, each company adopts different ports and protocols. Tridium fox is on port 1911 TCP. ETS (the commissioning tool for KNX networks) uses port 3671 UDP. There will be many more to add if we need to cater for all the products in a building. What is your thought on this: having specific commissioning MUD files for each tool or one that applies to all?